Qast/ViewControllers/MainViewController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  MainViewController.swift
+//  Qast
+//
+//  Created by Austin Welch on 7/19/19.
+//  Copyright © 2019 Qast. All rights reserved.
+//
+
+import BoseWearable
+import CoreLocation.CLLocation
+import UIKit
+
+class MainViewController: UIViewController {
+    
+    public lazy var mapViewController = MapViewController(networker: networker)
```

Comment:
By making this `lazy`, doesn't it ensure that `MapViewController` isn't initialized with `networker` until after `MainViewController` is initialized?

Reply:
Ah I see:

`init` sets `networker` -> `viewDidLoad` is called -> `mapViewController` is lazily loaded with the appropriate networker.

Awesome